modules/gitops-release-notes-1-7-1.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-7-1_{context}"]
+= Release notes for {gitops-title} 1.7.1
+
+{gitops-title} 1.7.1 is now available on {product-title} 4.8, 4.9, 4.10, 4.11, and 4.12.
+
+[id="errata-updates-1-7-1_{context}"]
+== Errata updates
+
+=== RHSA-2023:0467 - {gitops-title} 1.7.1 security update advisory 
+
+Issued: 2023-01-25
+
+The list of security fixes that are included in this release is documented in the link:https://access.redhat.com/errata/RHSA-2023:0467[RHSA-2023:0467] advisory. 
```

Comment:
Why is it different for 1.7.1

I see for 1.5.9 and 1.6.4, you mentioned the bugzilla issues. I think it is better to do the same thing here as well.

Reply:
@iam-veeramalla I have added a note in the PR description. From GitOps 1.7.0 onwards, we are going to use this new advisory fomat for all security fixes. To avoid any consistency issue with previous releases (1.6.4 and 1.5.9), we have used the same language that was there in the already published RN. The intent is to segregate security fixes from regular bug fixes.

Reply:
@Srivaralakshmi please note ^